CONTRIBUTING.md

```diff
@@ -18,4 +19,18 @@ In order to generate a `google-services.json` configuration file follow these st
 - Download `google-services.json` file.
 - Follow instructions to add file to project.
 - Skip "_Add Firebase SDK step_".
-- Run app to verify that the configuration is picked up correctly.
\ No newline at end of file
+- Run app to verify that the configuration is picked up correctly.
+
+## Translating
+If you want to help the project by translating the app, you have to:
+
+1. Go in [/app/src/main/res](https://github.com/ccomeaux/boardgamegeek4android/tree/develop/app/src/main/res);
```

Comment:
Let's make this a relative link, e.g. `app/src/main/res`.

Reply:
[This is a test](app/src/main/res)